open-api/rest-catalog-open-api.yaml

```diff
@@ -3324,6 +3348,217 @@ components:
           type: integer
           format: int64
 
+    BooleanTypeValue:
+      type: boolean
+      example: true
+
+    IntegerTypeValue:
+      type: integer
+      example: 42
+
+    LongTypeValue:
+      type: integer
+      format: int64
+      example: 9223372036854775807
+
+    FloatTypeValue:
+      type: number
+      format: float
+      example: 3.14
+
+    DoubleTypeValue:
+      type: number
+      format: double
+      example: 123.456
+
+    DecimalTypeValue:
+      type: string
+      description:
+        "Decimal type values are serialized as strings. Decimals with a positive scale serialize as numeric plain 
+        text, while decimals with a negative scale use scientific notation and the exponent will be equal to 
+        the negated scale"
+      example:
+        positiveScale: "123.4500"
+        zeroScale: "2"
+        negativeScale: "2E+20"
+
+    StringTypeValue:
+      type: string
+      example: "hello"
+
+    UUIDTypeValue:
+      type: string
+      format: uuid
+      description:
+        "UUID type values are serialized as a lowercase string"
+      example: "eb26bdb1-a1d8-4aa6-990e-da940875492c"
+
+    DateTypeValue:
+      type: string
+      format: date
+      description:
+        "Date type values follow the `YYYY-MM-DD` ISO-8601 standard date format"
+      example: "2007-12-03"
+
+    TimeTypeValue:
+      type: string
+      description:
+        "Time type values follow the `HH:MM[:SS[.sssssssss]]` ISO-8601 format with microsecond precision"
```

Comment:
nit: single quote instead of backquote

Reply:
We're adding nanoseconds, so we should state that both nanosecond and microsecond are allowed?